src/pages/Mypage/PageBody/PageBodyQna.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import variables from "styles/variables";
+import { AiOutlineArrowDown } from "react-icons/ai";
+import { AiOutlineArrowUp } from "react-icons/ai";
+
+const PageBodyQna = () => {
+  const [isMore, setIsMore] = useState(false);
+  const readMore = () => {
+    setIsMore((cur) => !cur);
+  };
+
+  return (
+    <ul>
+      <List onClick={readMore}>
+        <ListTitle>
+          <div>
+            <Title>개발자 채용 소식 문의합니다.</Title>
+            <Time>2022.10.14 문의 접수</Time>
+          </div>
+          <TitleRight>
+            <Text>답변 대기</Text>
+            {isMore ? (
+              <AiOutlineArrowDown size="18px" />
+            ) : (
+              <AiOutlineArrowUp size="18px" />
+            )}
+          </TitleRight>
+        </ListTitle>
+        {isMore ? null : (
+          <ListBody>개발자 채용공고는 어디서 볼 수 있나요?</ListBody>
+        )}
```

Comment:
```suggestion
               {!isMore && <ListBody>개발자 채용공고는 어디서 볼 수 있나요?</ListBody>}
```

여기도 삼항 연산자 대신 이렇게 수정할 수 있겠습니다!

Reply:
네 반영했습니다!